cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/SparkContextConfig.java

```diff
@@ -155,4 +168,8 @@ private void setLogicalStartTime(long startTime) {
   private void setTransaction(Transaction tx) {
     hConf.set(HCONF_ATTR_NEW_TX, GSON.toJson(tx));
   }
+
+  public void setWorkflowToken(WorkflowToken workflowToken) {
+    hConf.set(HCONF_ATTR_WORKFLOW_TOKEN, GSON.toJson(workflowToken));
```

Comment:
Will there be exception if token is null? Also, the parameter should have `@Nullable`.